README.md

```diff
@@ -216,7 +220,7 @@ Some caveats for Train loss and Test mAP:
 - In case you have tried a lot and still cannot get a loss around ~1.0 then add *one plot for all of your valid tries* and add it to the report. Also, add 2-3 lines on what you believe is the reason for the observed behavior.
 - Test AP (for detection) can show variance across different classes hence look at the mean value (mAP).
 
-Include all the code and images/logs after training.
+Include all the code and images/logs after training. Include appropriate comments on how you calculated class-wise AP and mAP.
```

Comment:
Also update this in submission checklist? Also, IIRC, the submission checklist doesn't match the question TODOs for task 2 in terms of logging frequencies/number of images to visualize, etc. Maybe we could standardize it?